pkg/ibmvpc/parser.go

```diff
@@ -776,7 +787,25 @@ func getSubnetByIPAddress(address string, c *vpcmodel.VPCConfig) (subnet *Subnet
 			}
 		}
 	}
-	return nil, fmt.Errorf("could not find matching subnet for address %s", address)
+	return nil, fmt.Errorf("could not find matching subnet for address %s", addressIPblock.ToIPAdressString())
+}
+
+func setNodeIPBlock(node vpcmodel.Node, address string) error {
+	addressIPBlock, err := common.NewIPBlockFromIPAddress(address)
+	if err != nil {
+		return err
+	}
+	switch nodeConcrete := node.(type) {
+	case *ReservedIP:
```

Comment:
because each case converts to a different type, and should access the field `ipblock` for each type separately (the input object is of interface `Node` that does not have this functionality).